libs/note-manager/src/note-manager.ts

```diff
@@ -450,11 +453,6 @@ export class NoteManager {
 
   /**
    * Generate a note
-   * @param sourceTypedChainId The source typed chain id
```

Comment:
Why remove this?

Reply:
Hey, @devpavan04, could you please assist me in reverting this documentation? I might have accidentally removed it. Please note that I removed the `tokenDecimals` argument.